<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWindow</class>
 <widget class="QMainWindow" name="StudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Potato Minus——学生界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SIMS_res.qrc">
    <normaloff>:/SIM_System_Pics/resources/icon_login_2.png</normaloff>:/SIM_System_Pics/resources/icon_login_2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="MsgBoard">
    <property name="geometry">
     <rect>
      <x>1221</x>
      <y>91</y>
      <width>301</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1221</x>
      <y>-9</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/SIM_System_Pics/resources/logo3.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{
	border:none;
}
QTabWidget::tab-bar{
        alignment:left;
}
QTabBar::tab{
	background:transparent;
	color:black;
	min-width:30ex;
	min-height:10ex;
}
QTabBar::tab:hover{
	background:rgb(255, 255, 255, 100);
}
QTabBar::tab:selected{
	border-color: white;
	background:white;
	color:green;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="TeacherTab">
     <attribute name="title">
      <string>我的课程</string>
     </attribute>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="FlushButton_1">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="DeleteLessonButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>退选课程</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="MyLessonWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>50</y>
        <width>1221</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>118</number>
      </attribute>
      <column>
       <property name="text">
        <string>课程编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程教师</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课院系</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学分</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课院系</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课年级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>上课时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程成绩</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="LessonTab">
     <attribute name="title">
      <string>课程管理</string>
     </attribute>
     <widget class="QTableWidget" name="LessonWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>1221</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>108</number>
      </attribute>
      <column>
       <property name="text">
        <string>课程编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程教师</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程最大人数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>当前选课人数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课院系</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学分</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课年级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开课学期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>上课时间</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="FlushButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ChooseLessonButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>选择课程</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ScholarTab">
     <attribute name="title">
      <string>奖学金申请</string>
     </attribute>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QToolButton" name="FlushButton_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ApplyScholarButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>申请奖学金</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="AppliableScholarWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>861</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="gridSize">
       <size>
        <width>1000</width>
        <height>40</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="AppliedScholarWidget">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>90</y>
        <width>361</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="gridSize">
       <size>
        <width>1000</width>
        <height>40</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;可申请的奖学金&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;申请结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>大创项目申请</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;申请结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QToolButton" name="FlushButton_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ApplyProButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;
hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;}</string>
         </property>
         <property name="text">
          <string>申请大创项目</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;可申请的大创项目&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="AppliableProWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>861</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="gridSize">
       <size>
        <width>1000</width>
        <height>40</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="AppliedProWidget">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>90</y>
        <width>361</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="gridSize">
       <size>
        <width>1000</width>
        <height>40</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="SIMS_res.qrc"/>
 </resources>
 <connections/>
</ui>
